src/client/components/PerjuryCheckbox/index.js

```diff
@@ -14,13 +15,20 @@ function PerjuryCheckbox(props) {
       className={isChecked ? "" : "unchecked"}
     >
       <Form.Text as="h3">{t("retrocerts-certification.ack-header")}</Form.Text>
-      <ul>
-        <li>{t("retrocerts-certification.ack-list-item-1")}</li>
-        <li>{t("retrocerts-certification.ack-list-item-2")}</li>
-        <li>{t("retrocerts-certification.ack-list-item-3")}</li>
-        <li>{t("retrocerts-certification.ack-list-item-4")}</li>
-      </ul>
-
+      {props.isPua ? (
```

Comment:
minor, mind doing a global find replace from isPua to isAnyWeekPua, to make that clearer?